src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -498,15 +499,20 @@ trait BronzeTransforms extends SparkSessionWrapper {
 
     // calling function to make parallel API calls
     val apiCallV2Obj = new ApiCallV2(config.apiEnv)
-    apiCallV2Obj.makeParallelApiCalls(clusterEventsEndpoint, jsonInput, config)
+    apiCallV2Obj.makeParallelApiCalls(clusterEventsEndpoint, jsonInput, pipelineSnapTime, config)
     logger.log(Level.INFO, " Cluster event landing completed")
   }
 
   private def processClusterEvents(tmpClusterEventsSuccessPath: String, organizationId: String, erroredBronzeEventsTarget: PipelineTable): DataFrame = {
     logger.log(Level.INFO, "COMPLETE: Cluster Events acquisition, building data")
     if (Helpers.pathExists(tmpClusterEventsSuccessPath)) {
+      //Code changes to get the raw response
       if (spark.read.json(tmpClusterEventsSuccessPath).columns.contains("events")) {
         try {
+          //Code changes to store the traceability DF
+          // TOMES - need to see how this makes it's way to target
+          // TOMES -- add pipeline_target
+          createTraceabilityDF(spark.read.json(tmpClusterEventsSuccessPath))
```

Comment:
remove all references to the workd `traceability` this one should be something like `deriveAPIMetaDF` or something